backend/tests/services/common/DeploymentFrequency.test.ts

```diff
@@ -145,6 +145,7 @@ describe("DeploymentFrequency", () => {
   );
 
   before(async function () {
+    this.timeout(10000);
```

Comment:
Is there any blocking func that will make it longer than 10s? should we consider mocking it?

Reply:
removed that, it no use at all.